src/WebHelpers.bas

```diff
@@ -1634,7 +1634,7 @@ Public Function ExecuteInShell(web_Command As String) As ShellResult
 
 web_Cleanup:
 
-    ExecuteInShell.ExitCode = CInt(web_pclose(web_File))
+    ExecuteInShell.ExitCode = CLng(web_pclose(web_File))
```

Comment:
It would be ideal to refactor `ExecuteInShell.ExitCode` (and `utc_ExecuteInShell.utc_ExitCode`) to be `LongPtr` on `Mac/VBA7`, but this gets the job done. It's not likely that `pclose` would return something that exceeds the capacity of `Long` in any reasonable scenario, but since it returns `size_t` which resolves as `LongLong` on 64-bit systems, it may be worth it to make that change.